bcda/service/service.go

```diff
@@ -359,6 +359,23 @@ func (s *service) getNewAndExistingBeneficiaries(ctx context.Context, conditions
 		return nil, nil, CCLFNotFoundError{8, conditions.CMSID, conditions.fileType, cutoffTime}
 	}
 
+	// If the _since parameter is more recent than the latest CCLF file processing date,
+	// all beneficiaries should be considered pre-existing benes.
+	if !conditions.Since.IsZero() && cclfFileNew.CreatedAt.Sub(conditions.Since) < 0 {
+		// Retrieve all of the benes associated with this CCLF file.
+		benes, err := s.getBenesByFileID(ctx, cclfFileNew.ID, conditions)
+		if err != nil {
+			return nil, nil, err
+		}
+
+		if len(benes) == 0 {
+			return nil, nil, fmt.Errorf("Found 0 new or existing beneficiaries from CCLF8 file for cmsID %s cclfFiledID %d",
+				conditions.CMSID, cclfFileNew.ID)
+		}
+
+		return newBeneficiaries, benes, nil
```

Comment:
Style question, since newBeneficiaries in this instance would be 0 / newBeneficiaries isn't assigned, what are your thoughts on returning 0 in lieu of newBenes as a variable?

Reply:
Got it! Definitely don't want to seg fault, especially in a language that's not C.